src/generate_test_vectors.py

```diff
@@ -810,15 +810,26 @@ def formatResults(sig, s ):
 
   cert = signSigCert(sig)
   jsonTest['x5c'] = base64.b64encode(cert.public_bytes(encoding=serialization.Encoding.DER)).decode('ascii')
-  jsonTest['sk'] = base64.b64encode(sig.private_key_bytes()).decode('ascii')
+
+  # for standalone ML-DSA, we need to wrap the private key in an OCTET STRING, but not when it's a composite
+  if sig.id in ("id-ML-DSA-44", "id-ML-DSA-65", "id-ML-DSA-87"):
+    jsonTest['sk'] = base64.b64encode( 
+                  encode(univ.OctetString(sig.private_key_bytes()))
+                                                      ).decode('ascii')
+  else:
+    jsonTest['sk'] = base64.b64encode(sig.private_key_bytes()).decode('ascii')
 
     # Construct PKCS#8
   pki = rfc5208.PrivateKeyInfo()
   pki['version'] = 0
   algId = rfc5208.AlgorithmIdentifier()
   algId['algorithm'] = OID_TABLE[sig.id]
   pki['privateKeyAlgorithm'] = algId
-  pki['privateKey'] = univ.OctetString(sig.private_key_bytes())
+  # for standalone ML-DSA, we need to wrap the private key in an OCTET STRING, but not when it's a composite
+  if sig.id in ("id-ML-DSA-44", "id-ML-DSA-65", "id-ML-DSA-87"):
+    pki['privateKey'] = univ.OctetString(univ.OctetString(sig.private_key_bytes()))
```

Comment:
This is encoding the seed as a `seed OCTET STRING` but really it needs to be `seed [0] IMPLICIT OCTET STRING`

```suggestion
    pki['privateKey'] = univ.OctetString(encode(univ.OctetString(sig.private_key_bytes()).subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 0))))
```

And if you look up, way up, you'll have make the following change. github won't let me make a suggestion up there.
```
from pyasn1.type import univ
```
to
```
from pyasn1.type import univ, tag
```